app/views/vendors/index.html.erb

```diff
@@ -1,76 +1,38 @@
 <div class="row">
-  <% @vendors.each do |vendor| %>
-
   <div class="large-4 small-12 columns">
 
     <img src="http://placehold.it/500x500&text=Logo">
 
     <div class="hide-for-small panel">
```

Comment:
Yup, exactly! Anything with the class `hide-for-small` will be hidden on small screens, along with all the stuff inside it.